quantuminspire/util/configuration.py

```diff
@@ -150,6 +142,18 @@ def settings_customise_sources(  # pylint: disable=R0913
             file_secret_settings,
         )
 
+    async def fetch_suggested_auth_settings(self, host: Optional[Url] = None) -> None:
```

Comment:
It's not really meant as "suggested". These are the correct settings for this API. However, if you really really have a different opinion...

Reply:
Agree, changed to `fetch_auth_settings` instead